### ISSUE TITLE
Validate email

### ISSUE THREAD

**Issue by amaliaibarra** [josejorgers/to-the-rescue#16]
Validate if the email provided when a user registers is valid.

**Comment by svege** [comment]
Hello, can I take this issue?

**Comment by amaliaibarra** [comment]
Hi @svege!!! Yes! You can take this issue. I edited it to be a little more specific. If you have any questions feel free to ask 😊. Thank you very much for chosing our project to collaborate!! 🎉

**Comment by amaliaibarra** [comment]
Let me know if you're still interested after the edit so I can assign it to you :)

**Comment by svege** [comment]
Yes, I'd like to do it! After research of the issue, I agree that confirmation by email would be the best choice.

**Comment by josejorgers** [comment]
Great! It's all yours @svege. Thanks for your interest in this project
